source/scss/global.scss

```diff
@@ -62,11 +62,16 @@ h5 {
 }
 
 a {
-    color: $primary_dark_90;
+    font: $font_headings;
```

Comment:
Move this to the header nav code, normal content links should be in the same font as the rest of the text

Reply:
done